ui/raidboss/data/07-dt/raid/r2s.ts

```diff
@@ -105,6 +105,18 @@ const triggerSet: TriggerSet<Data> = {
       suppressSeconds: 5,
       response: Responses.spread(),
     },
+    {
+      id: 'R2S Headmarker Alarm Pheromones Puddle',
+      type: 'HeadMarker',
+      netRegex: { id: headMarkerData.spreadMarker1, capture: true },
+      condition: Conditions.targetIsYou(),
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Drop Puddle',
```

Comment:
```suggestion
          en: 'Drop Puddle Outside',
```

There's not really another way to resolve this mechanic properly, so the callout should include the `outside` instruction.

Reply:
Seems like this might be a useful thing to stick in resources too